AEPLifecycle/Sources/Lifecycle.swift

```diff
@@ -82,7 +82,7 @@ public class Lifecycle: NSObject, Extension {
         let prevSessionInfo = lifecycleState.start(date: event.timestamp,
                                                    additionalContextData: event.additionalData,
                                                    adId: getAdvertisingIdentifier(event: event),
-                                                   sessionTimeout: getSessionTimeoutLength(configurationSharedState: configurationSharedState.value))
+                                                   sessionTimeout: getSessionTimeoutLength(configurationSharedState: configurationSharedState.value), dispatchApplicationLaunchEvent: dispatchApplicationLaunch(xdm:data:))
```

Comment:
I would prefer not to escape the dispatchers to LifecycleState. I seems like we need to update LifecycleState to allow for fetching regular vs xdm content so we can dispatch the two events from here.
This way we can also update the order of the events, Lifecycle Response Content first, then Lifecycle Application Launch.